core/lib/makeswift/components/site-footer/site-footer.client.tsx

```diff
@@ -63,21 +65,18 @@ function combineSections(
   );
 }
 
-export const MakeswiftFooter = forwardRef(
```

Comment:
Removed `forwardRef` as the Soul's `Footer` component does not forward `ref` anymore (which is okay since it was previously simply forwarding the ref to the root element).

Reply:
The diffs are best reviewed with "Hide whitespace" toggle on.